server.go

```diff
@@ -240,6 +239,16 @@ func (s *Server) CreateHandler() http.Handler {
 	return s.rootHandler(s.AroundRequest(s.requestHandler()))
 }
 
+func (s *Server) createResponseHandler() http.Handler {
```

Comment:
Should this be made public, or is there a way we can ensure `createResponseHandler` and `CreateResponseHandler` are called when testing? Just thinking of how consumers of the package can test viewproxy without having to listen on a port, if we want to keep supporting that.

Reply:
Thanks for raising this, I realized this should be created/used from `requestHandler` which is called from `CreateHandler` so it removes the need to set it up or store it on the serve struct.